### DISCUSSION TITLE
AquaPi Cam STL

### DISCUSSION THREAD

**Discussion by kiguy2052** [TheRealFalseReality/aquapi#73] [Q&A] [answered]
Does anyone have the STLs for the camera case? I've yet to find a better one anywhere online, and it's missing from the STLs[here](https://github.com/TheRealFalseReality/aquapi/tree/main/assests/STLModels)

**Comment by TheRealFalseReality** [comment, marked as answer]
Yep! Just added to the same folder. I also split the repository for the SnapCam, you can also find those [here](https://github.com/TheRealFalseReality/esp32_snapcam/tree/main/assests/STLModels/SnapCam)

**Reply by TheRealFalseReality** [reply]
Tip: Use zip ties around the ball joint to get a tighter fit

**Reply by TheRealFalseReality** [reply]
Also posted on [Thingiverse](https://www.thingiverse.com/thing:6716128)
